apps/remix-ide/src/app/udapp/make-udapp.js

```diff
@@ -2,7 +2,6 @@ var registry = require('../../global/registry')
 var remixLib = require('@remix-project/remix-lib')
 var yo = require('yo-yo')
 var EventsDecoder = remixLib.execution.EventsDecoder
-var TransactionReceiptResolver = require('../../lib/transactionReceiptResolver')
```

Comment:
`transactionReceiptResolver.js` the file seems not removed, do you still need it?